study-builder/studies/pancan/study-workflows.conf

```diff
@@ -27,39 +27,66 @@
         {
           "type": "STUDY_REDIRECT",
           "studyGuid": "cmi-mbc",
+          "studyName": "Metastatic Breast Cancer"
```

Comment:
For these that are Pepper studies, I recommend not setting `studyName` so we use the other configurations (and also let us know the code for looking it up is working!). Also, `Metastatic Breast Cancer` is not exactly what we want. I think we should add `Project` at the end there. Again, I recommend not setting it and use our other configurations.

Reply:
I thought now that we have more flexibility, we can pass exactly what we need for front-end to show. Avoid cases where like "ascproject" is passed. 
Any way will remove and we can revisit if needed.

Reply:
We have ticket to clean up those names: https://broadinstitute.atlassian.net/browse/DDP-6329.